### ISSUE TITLE
release test 2018.01 RC1: tests/thread_priority_inversion, hangup

### ISSUE THREAD

**Issue by A-Paul** [RIOT-OS/RIOT#8444]
#### Description
Sequence of messages stop without producing any error message.

#### Steps to reproduce the issue
"make clean all term" in *tests/thread_priority_inversion*

#### Expected results
According to the README.md

```
...
2017-07-17 17:00:31,335 - INFO # t_mid: doing some stupid stuff...
2017-07-17 17:00:31,340 - INFO # t_high: allocating resource...
```

#### Actual results
> main(): This is RIOT! (Version: 2018.04-devel-17-ga78b-mobi31-2018.01-branch)
> This is a scheduling test for Priority Inversion
> t_low: allocating resource...
> t_low: got resource.
> t_high: allocating resource...
> t_low: freeing resource...
> t_high: got resource.
> t_low: freed resource.
> t_high: freeing resource...
> t_high: freed resource.
> t_low: allocating resource...
> t_low: got resource.
> t_high: allocating resource...
> t_mid: doing some stupid stuff...

hangs here ...

#### Versions
```
----------------------------
          native gcc: gcc (Debian 6.3.0-18) 6.3.0 20170516
          msp430-gcc: msp430-gcc (GCC) 4.6.3 20120301 (mspgcc LTS 20120406 unpatched)
             avr-gcc: avr-gcc (GCC) 4.9.2
   arm-none-eabi-gcc: arm-none-eabi-gcc (15:5.4.1+svn241155-1) 5.4.1 20160919
    mips-mti-elf-gcc: missing/error
../../dist/tools/ci/print_toolchain_versions.sh: 92: ../../dist/tools/ci/print_toolchain_versions.sh: clang: not found
arm-none-eabi-newlib: "2.4.0"
 mips-mti-elf-newlib: missing/error
            avr-libc: "1.8.0svn" ("20111229")
            cppcheck: missing
          coccinelle: missing
                 git: git version 2.11.0
```

**Comment by smlng** [comment]
known issue, fix under discussion in #7445

**Comment by cladmi** [comment]
This one should have a dedicated automated test I think. It would make it more visible.

**Comment by PeterKietzmann** [comment]
Duplicate of #7365. Fix (not yet merged) in #7445
